app/Providers/AuthServiceProvider.php

```diff
@@ -76,5 +77,14 @@ class AuthServiceProvider extends ServiceProvider
     public function boot(): void
     {
         $this->registerPolicies();
+
+        if (! $this->app->routesAreCached()) {
+            Passport::routes();
+        }
+
+        Passport::hashClientSecrets();
+        Passport::tokensExpireIn(now()->addDay());
+        Passport::refreshTokensExpireIn(now()->addMonth());
+        Passport::personalAccessTokensExpireIn(now()->addYear());
```

Comment:
Add custom model to skip auth screen - https://laravel.com/docs/8.x/passport#approving-the-request